rotkehlchen/api/rest.py

```diff
@@ -854,13 +854,14 @@ def edit_trade(
             result_dict = _wrap_in_ok_result(result_dict)
             return api_response(result_dict, status_code=HTTPStatus.OK)
 
-    def delete_trade(self, trade_id: str) -> Response:
-        with self.rotkehlchen.data.db.user_write() as cursor:
-            result, msg = self.rotkehlchen.data.db.delete_trade(cursor, trade_id)
-        if not result:
-            return api_response(wrap_in_fail_result(msg), status_code=HTTPStatus.CONFLICT)
-
-        return api_response(_wrap_in_ok_result(True), status_code=HTTPStatus.OK)
+    def delete_trades(self, trades_ids: List[str]) -> Response:
+        try:
+            with self.rotkehlchen.data.db.user_write() as cursor:
+                self.rotkehlchen.data.db.delete_trades(cursor, trades_ids)
+        except InputError as e:
+            return api_response(wrap_in_fail_result(str(e)), status_code=HTTPStatus.CONFLICT)
+        else:
+            return api_response(_wrap_in_ok_result(True), status_code=HTTPStatus.OK)
```

Comment:
What is the advantage of using the else vs just
```suggestion
    return api_response(_wrap_in_ok_result(True), status_code=HTTPStatus.OK)
```

I would use the latest since it is what we do in other parts of the code